README.md

```diff
@@ -10,14 +10,22 @@ For guides on installing, using and configuring CoffeeLint, head over
 To suggest a feature, report a bug, or general discussion, head over
 [here](http://github.com/clutchski/coffeelint/issues/).
 
-## Team
+## :rotating_light: Team :rotating_light:
+
+There are no future plans for CoffeeLint. Only major bugs will be addressed
```

Comment:
I don't think I want to put out a call for a new maintainer. I've seen people come and go over the years and advancing this project is actually a lot of work. I'd want to see how they intend to advance the project before trying to hand anything over though. If they're passionate enough about the project I'm assuming they'll ignore the warning and contact me about it.

I actually don't have full control of the project. @clutchski gave me commit access after I made a series of PRs a few years ago. If someone wants commit access, we still need to contact him about it.